src/node.rs

```diff
@@ -365,26 +373,55 @@ impl InMemoryNode {
         push_transaction_to_bootloader_memory(&mut vm, &tx, execution_mode, None);
         let tx_result = vm.execute_next_tx(u32::MAX, true).unwrap();
 
+        println!("\n=======================");
+        println!("TRANSACTION SUMMARY");
+        println!("=======================\n");
+
         match tx_result.status {
             TxExecutionStatus::Success => println!("Transaction: {}", "SUCCESS".green()),
             TxExecutionStatus::Failure => println!("Transaction: {}", "FAILED".red()),
         }
 
         println!(
-            "Initiator: {:?} Payer: {:?}",
+            "Initiator: {:?}\nPayer: {:?}",
             tx.initiator_account(),
             tx.payer()
         );
-
         println!(
-            "{} Limit: {:?} used: {:?} refunded: {:?}",
-            "Gas".bold(),
+            "Gas - Limit: {:?} | Used: {:?} | Refunded: {:?}",
```

Comment:
Nice 👍 

I also think we could start emulating the UI from other simulation nodes, like in Tenderly:
![image](https://github.com/matter-labs/era-test-node/assets/1890113/42619bdc-42b0-4e66-8851-d7f5c821f355)

Perhaps these could be a set of `good first issue 🐣 ` Github Issues?